jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -281,18 +285,13 @@ public void invoke(int error, long description) {
         }
 
         final GLFWVidMode videoMode = glfwGetVideoMode(glfwGetPrimaryMonitor());
-
-        if (settings.getWindowWidth() <= 0 || settings.getWindowHeight() <= 0) {
-            settings.setWindowSize(videoMode.width(), videoMode.height());
-        } else {
-            settings.setWindowSize(settings.getWindowWidth(), settings.getWindowHeight());
+        int requestWidth = settings.getWindowWidth();
+        int requestHeight = settings.getWindowHeight();
+        if (requestWidth <= 0 || requestWidth <= 0) {
```

Comment:
`if (requestWidth <= 0 || requestWidth <= 0)`

I think should be 

 `if (requestWidth <= 0 || requestHeight <= 0)`

Reply:
Fixed in #1950

Reply:
Good catch!